bin/benchmark_elbo.jl

```diff
@@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat")
 srand(1)
 println("Loading data.")
 
-S = 100
+S = 3
```

Comment:
Sure, done.  However, you should know that it is not as fast -- it's perhaps 50% slower.  I spent a fair amount of time trying to get it back to the original speed and am out of ideas.  After I submit, it would be great if you took a pass.  You have a lot more insight than I do into optimizing julia code.